src/mscorlib/src/System/AggregateException.cs

```diff
@@ -29,7 +29,9 @@ namespace System
     /// <see cref="AggregateException"/> is used to consolidate multiple failures into a single, throwable
     /// exception object.
     /// </remarks>
+#if FEATURE_SERIALIZATION
     [Serializable]
```

Comment:
none of these are actually visible yet right because we don't have "serializableattribute" in model.xml ?